src/latinize.ts

```diff
@@ -4,6 +4,7 @@ import { normalizeSymbols, SymbolOptions } from "./normalizeSymbols";
 
 export interface LatinizeOptions extends SymbolOptions, DiacriticOptions {
   symbols?: boolean;
+  removeDiacritics?: boolean;
```

Comment:
I don't see this option in the code and I don't think it makes sense here? `latinize` is supposed to entirely ascii-fold the string, thus removing diacritics is required